snapcraft/plugins/_python/_pip.py

```diff
@@ -142,34 +163,50 @@ def setup(self):
             try:
                 self._python_home = os.path.join(os.path.sep, 'usr')
 
-                # Using the host's pip, install our own pip and other tools we
-                # need.
-                self.download({'pip', 'setuptools', 'wheel'})
-                self.install(
-                    {'pip', 'setuptools', 'wheel'}, ignore_installed=True)
+                # Using the host's pip, install our own pip
+                self.download({'pip'})
+                self.install({'pip'}, ignore_installed=True)
             finally:
                 # Now that we have our own pip, reset the python home
                 self._python_home = real_python_home
 
-    def is_setup(self):
-        """Return true if this class has already been setup."""
+    def _ensure_wheel_installed(self):
+        if not self._is_wheel_installed():
+            logger.info('Fetching wheel...')
```

Comment:
same comment as the one for the pip log. This fetches and installs.

Reply:
Fixed.